packages/terra-application-navigation/src/common/_PopupCount.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { useAnimatedCount } from '../utils/helpers';
+
+import styles from './PopupCount.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The number of notifications to display.
+   */
+  value: PropTypes.number,
+};
+
+const PopupCount = ({ value }) => {
+  const countRef = useRef();
+
+  useAnimatedCount(countRef, value);
+
+  let validatedValue = value;
+  if (value > 999) {
```

Comment:
This is totally nit picky on my part, but 999+ to me says 999 or greater.  So the check should be value >= 999.  I don't really care necessarily

Reply:
![](https://media.giphy.com/media/1hMk0bfsSrG32Nhd5K/giphy.gif)